vendor/engines/sanger_sequencing/config/locales/en.primers.yml

```diff
@@ -0,0 +1,7 @@
+en:
+  sanger_sequencing:
+    primer:
+      default_list:
+        - M13FM13RT7
+        - PromoterT7
+        - Terminator
```

Comment:
It may be a good idea to confirm these primers should be the default ones. I know you said these are the ones you saw, is it for both Dartmouth and OSU?

Reply:
I'll ask